apps/studio/src/constants/formBuilder.ts

```diff
@@ -11,7 +11,7 @@ export const JSON_FORMS_RANKING = {
   ObjectControl: 2,
   AllOfControl: 3,
   AnyOfControl: 3,
-  ProseControl: 2,
+  ProseControl: 3,
```

Comment:
question - can i check what this fixes?

Reply:
it might be matched by `jsonFormsObjectControlTester` otherwise since both are `2`, so setting `3` will ensure it's checked first before that